packages/web-react/src/components/TextArea/__tests__/TextArea.test.tsx

```diff
@@ -45,12 +45,19 @@ describe('TextArea', () => {
   });
 
   it('should have message', () => {
-    const dom = render(<TextArea id="textarea" label="Label" message="text" />);
+    const dom = render(<TextArea id="textarea" label="Label" message="text" validationState="danger" />);
```

Comment:
maybe we should test it without `validationState` too?

Reply:
ok